articles/confidential-computing/overview.md

```diff
@@ -90,11 +90,11 @@ An application built with enclaves is partitioned in two ways:
 1. An "untrusted" component (the host)
 1. A "trusted" component (the enclave)
 
-**The host** is your enclave application running on an untrusted environment. The code in the host can't access the code loaded into the enclave. 
+**The host** is where your enclave application is running on top and is an untrusted environment. The enclave code deployed on the host can't be accessed by the host. 
```

Comment:
This sentence seems to read wrong. Maybe "the host is where your enclave application is running on top **of** and is an untrusted environment"

Reply:
Host is untrusted but the part of the application can start an enclave which is secured.